### PR TITLE
Fix duplicated atom key

### PR DESCRIPTION
**Pull request by yusukefs**
## What?

atomのキーで被っていたものがあったため変更

## Why?

以下の警告が出ていた

```
Duplicate atom key "recordPaginateState". This is a FATAL ERROR in
      production. But it is safe to ignore this warning if it occurred because of
      hot module replacement.
```

### CODE REVIEW

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/82#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by WatanabeToshimitsu — APPROVED** [review]
あざます！

**Comment by hdl-service** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/82#pullrequestreview-707005339" title="Approved">WatanabeToshimitsu</a>*, *<a href="https://github.com/dataware-tools/app-data-browser-next/pull/82#" title="Author self-approved">yusukefs</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=dataware-tools%2Fapp-data-browser-next).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/dataware-tools/app-data-browser-next/blob/master/OWNERS)~~ [WatanabeToshimitsu,yusukefs]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
